cmd/posix.go

```diff
@@ -101,7 +101,7 @@ func newPosix(diskPath string) (StorageAPI, error) {
 	}
 	fs := &posix{
 		diskPath:    diskPath,
-		minFreeDisk: fsMinSpacePercent, // Minimum 5% disk should be free.
+		minFreeDisk: fsMinFreeSpace,
```

Comment:
This logic needs to change since the previous code is using percent, so it won't work magically.